interface/super/manage_document_templates.php

```diff
@@ -129,97 +129,93 @@
 
 ?>
 <html>
-
-<head>
-<title><?php echo xlt('Document Template Management'); ?></title>
-<?php Header::setupHeader(); ?>
-
-<style type="text/css">
- .dehead { color:var(--black); font-family:sans-serif; font-size:10pt; font-weight:bold }
- .detail { color:var(--black); font-family:sans-serif; font-size:10pt; font-weight:normal }
-</style>
-
-</head>
-
-<body class="body_top">
-<form method='post' action='manage_document_templates.php' enctype='multipart/form-data'
- onsubmit='return top.restoreSession()'>
-<input type="hidden" name="csrf_token_form" value="<?php echo attr(CsrfUtils::collectCsrfToken()); ?>" />
-
-<center>
-
-<h2><?php echo xlt('Document Template Management'); ?></h2>
-
-<p>
-<table border='1' width='95%'>
-
- <tr bgcolor='#dddddd' class='dehead'>
-  <td align='center'><?php echo xlt('Upload a Template'); ?></td>
- </tr>
-
- <tr>
-  <td valign='top' class='detail' style='padding:10pt;' nowrap>
-    <?php echo xlt('Source File'); ?>:
-   <input type="hidden" name="MAX_FILE_SIZE" value="250000000" />
-   <input type="file" name="form_file" size="40" />&nbsp;
-    <?php echo xlt('Destination Filename'); ?>:
-   <input type='text' name='form_dest_filename' size='30' />
-   &nbsp;
-   <input type='submit' name='bn_upload' value='<?php echo xla('Upload') ?>' />
-  </td>
- </tr>
-
-</table>
-</p>
-
-<p>
-<table border='1' width='95%'>
-
- <tr bgcolor='#dddddd' class='dehead'>
-  <td align='center'><?php echo xlt('Download or Delete a Template'); ?></td>
- </tr>
-
- <tr>
-  <td valign='top' class='detail' style='padding:10pt;' nowrap>
-   <select name='form_filename'>
-<?php
-// Generate an <option> for each existing file.
-if (file_exists($templatedir)) {
-    $dh = opendir($templatedir);
-} else {
-    $dh = false;
-}
-if ($dh) {
-    $templateslist = array();
-    while (false !== ($sfname = readdir($dh))) {
-        if (substr($sfname, 0, 1) == '.') {
-            continue;
-        }
-
-        $templateslist[$sfname] = $sfname;
-    }
-
-    closedir($dh);
-    ksort($templateslist);
-    foreach ($templateslist as $sfname) {
-        echo "    <option value='" . attr($sfname) . "'";
-        echo ">" . text($sfname) . "</option>\n";
-    }
-}
-?>
-   </select>
-   &nbsp;
-   <input type='submit' name='bn_download' value='<?php echo xla('Download') ?>' />
-   &nbsp;
-   <input type='submit' name='bn_delete' value='<?php echo xla('Delete') ?>' />
-  </td>
- </tr>
-
-</table>
-</p>
-
-</center>
-
-</form>
-</body>
-</html>
+   <head>
+      <title><?php echo xlt('Document Template Management'); ?></title>
+      <?php Header::setupHeader(); ?>
+      <style type="text/css">
+         .dehead { color:var(--black); font-family:sans-serif; font-size:10pt; font-weight:bold }
+         .detail { color:var(--black); font-family:sans-serif; font-size:10pt; font-weight:normal }
+      </style>
+   </head>
+   <body class="body_top">
+      <form method='post' action='manage_document_templates.php' enctype='multipart/form-data'
```

Comment:
I think we need to add the container class.

Reply:
agreed